test/e2e/run-multiple.sh

```diff
@@ -28,8 +28,8 @@ for MANIFEST in "$@"; do
 		echo "==> Dumping container logs for $MANIFEST..."
 		./build/runner -f "$MANIFEST" logs
 
-		echo "==> Cleaning up failed testnet $MANIFEST..."
-		./build/runner -f "$MANIFEST" cleanup
+	#	echo "==> Cleaning up failed testnet $MANIFEST..."
+	#	./build/runner -f "$MANIFEST" cleanup
```

Comment:
Why's this commented out?

Reply:
This should not have been pushed, must have mistakenly staged it, this file should not be changed.